domain/ide/codeaction/codeaction.go

```diff
@@ -1,35 +0,0 @@
-/*
- * © 2022 Snyk Limited All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package codeaction
```

Comment:
Why was it moved to application instead of remaining in domain?

Reply:
Because we're using `lsp` types which are defined in the application layer